esm/svg/element.js

```diff
@@ -17,7 +18,7 @@ const handler = {
  */
 export class SVGElement extends Element {
   constructor(ownerDocument, localName, ownerSVGElement = null) {
-    super(ownerDocument, localName);
+    super(ownerDocument, localName, SVG_NAMESPACE);
```

Comment:
every `SVGElement` element has implicitly `SVG_NAMESPACE` as `namespaceURI` ... I feel like these elements are the only one that should have something meaningful in here.

Reply:
Amend, I am OK to have either lazy results or specific `get namespaceURI` for SVG elements that override the default for HTML one.